universal-application-tool-0.0.1/app/views/questiontypes/NumberQuestionRenderer.java

```diff
@@ -0,0 +1,34 @@
+package views.questiontypes;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.input;
+import static j2html.TagCreator.label;
+import static j2html.TagCreator.text;
+
+import j2html.tags.Tag;
+import services.applicant.ApplicantQuestion;
+import views.BaseHtmlView;
+
+public class NumberQuestionRenderer extends BaseHtmlView implements ApplicantQuestionRenderer {
+
+  private final ApplicantQuestion question;
+
+  public NumberQuestionRenderer(ApplicantQuestion question) {
+    this.question = checkNotNull(question);
+  }
+
+  @Override
+  public Tag render() {
+    ApplicantQuestion.NumberQuestion numberQuestion = question.getNumberQuestion();
+
+    return label(
+        text(question.getQuestionText()),
```

Comment:
Thoughts(non-blocking): Aaron has a PR that enhances existing question types, might worthing checkout out there (and maybe matching the styles).

Reply:
Awesome, I hadn't seen that yet! I'll check it out

Reply:
Updated to match `TextQuestionRenderer` - thanks for the heads up!